x/inflation/keeper/inflation_test.go

```diff
@@ -137,7 +137,7 @@ func TestGetCirculatingSupplyAndInflationRate(t *testing.T) {
 	}{
 		{
 			"no epochs per period",
-			sdk.TokensFromConsensusPower(400_000_000, sdk.DefaultPowerReduction),
+			sdk.TokensFromConsensusPower(400_000_000-100_000_001, sdk.DefaultPowerReduction),
```

Comment:
Have to account for the new tokens from genesis state that were changed from `stake` to `unibi`.